hikey.mk

```diff
@@ -278,10 +278,10 @@ filelist-all: busybox
 	export TOP=$(ROOT); export MULTIARCH=$(MULTIARCH); \
 	$(expand-env-var) <$(PATCHES_PATH)/rootfs/initramfs-add-files.txt >> $(GEN_ROOTFS_PATH)/filelist-all.txt; \
```

Comment:
As far as running commands with specific variables set, might it not be clearer to use the `env` command:
```
        env TOP=$(ROOT) MULTIARCH=$(MULTIARCH) command...
```